react/Components/Footers/FooterDesktop.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react'
+import style from "./FooterDesktop.css"
+
+interface FooterProps {
+
+}
+
+const FooterDesktop: StorefrontFunctionComponent<FooterProps> = (props: any) => {
+    const footerSectionItems = [
+        {
+            title: "Shop",
+            items: props.shopLinks ? props.shopLinks : []
+        },
+        { title: "About", items: props.aboutLinks ? props.aboutLinks : [] },
+        { title: "Help", items: props.helpLinks ? props.helpLinks : [] }
+    ]
+    console.log(footerSectionItems);
+
+    return (<div>
+        <div className={`${style.title} ph7 pt4`}>
+            {
+                footerSectionItems.map(ele => (
+                    <div key={ele.title} >
+                        <b>{ele.title}</b>
+                        <div>
+                            {ele.items.length && ele.items.map((menu: any) => (
+                                <p key={menu.text}>{menu.text}</p>
+                            ))}
+                        </div>
+                    </div>
+
+                ))
+
+            }
+        </div>
+        <div className={`${style.title} ph7 pt3`}>
+            <div className="flex flex-column">
+                <p className='b '>Sign up to receive the project antelope newsletter</p>
+                <input type='email' className={`${style.email}`} placeholder='Enter your email address' />
+                <span className={`${style.arrow}`}> <svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 448 512">
+                    <path d="M438.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-160-160c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3L338.8 224 32 224c-17.7 0-32 14.3-32 32s14.3 32 32 32l306.7 0L233.4 393.4c-12.5 12.5-12.5 32.8 0 45.3s32.8 12.5 45.3 0l160-160z"></path></svg>
+                </span>
+            </div>
+            <div className="">
+                <p className={`${style.heading}b flex justify-end`}>Connect with us</p>
+                <div className=' flex justify-end'>
+                    <div className='ph3'>
+                        <svg xmlns="http://www.w3.org/2000/svg" height="2em" viewBox="0 0 448 512">
```

Comment:
use vtex style guide for icons and please refer to the doc: [https://styleguide.vtex.com/#/Icons](url)

Ex: import { Plus } from 'vtex.styleguide'